### ISSUE TITLE
New-Event does not provide new item

### ISSUE THREAD

**Issue by stefanuebe** [vaadin/vaadin-crud-flow#204]
I wanted to initialize a new item for the CRUD editor with some default values. To do that, I used `event.getItem()`. This method returns `null`, which makes somehow sense, since it is a new item, but on the other hand makes this method completely useless plus I'm not able to initialize the new item at all.

Request:  NewEvent.getItem() should return the item, that will be passed to the binder afterwards to allow custom initialization.

**Comment by KasparScherrer** [comment]
I noticed this too and was very confused. 

my workaround: adding my own new-button that prepares my new item and then calling `crud.edit(preparedItem, NEW_ITEM)`, and don't display the default add-button by setting the cruds toolbar to an empty Span.  

I found that I need to prepare my new items more often than not. Having a NewEvent Listener that actually has an item would be very welcome to me.

**Comment by TatuLund** [comment]
> my workaround: adding my own new-button that prepares my new item and then calling crud.edit(preparedItem, NEW_ITEM),

Yes, this works because new item is set to fired event in edit method, but not when clicking new button of the CRUD.

**Comment by TatuLund** [comment]
I added fix for this in Enhanced CRUD https://vaadin.com/directory/component/enhanced-crud

**Comment by KasparScherrer** [comment]
Thanks Tatu. This add-on is looking neat, I had no idea it exists. Will definitely check it out right now, it covers several issues I had with crud.
